javascript/javascript2/week1/spiritanimalgenerator.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    
+
+    <label for="name">Name :</label>
+    <input type="text" id="name" placeholder="Enter name" required
+       minlength="4"  size="20">
+    <button type="button" id="myButton" >Click Here</button>
+  
+    <div id ="animal"></div>
+
+
+    <script type ="text/javascript">
+        const spiritAnimal = ["Keeper Armadillo","Guardian Bat","Gentle Deer","Independent Badger","Lawkeeper Crow","Loyal Dog","Friendly Dolphin","Swift Horse","The Fullmoon wolf","Crying butterfly"];
+
+        const buttonTag = document.getElementById('myButton')
+
+		
+		buttonTag.addEventListener('click', function(event){
```

Comment:
you should use the Format Document option in VSCode to clean this up, it can be hard to tell what is going on if the indentation is a bit funny

Reply:
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Document</title>
  </head>
  <body>
    <label for="name">Name :</label>
    <input
      type="text"
      id="name"
      placeholder="Enter name"
      required
      minlength="4"
      size="20"
    />
    <button type="button" id="myButton">Click Here</button>

    <div id="animal"></div>

    <script type="text/javascript">
      const spiritAnimal = [
        "Keeper Armadillo",
        "Guardian Bat",
        "Gentle Deer",
        "Independent Badger",
        "Lawkeeper Crow",
        "Loyal Dog",
        "Friendly Dolphin",
        "Swift Horse",
        "The Fullmoon wolf",
        "Crying butterfly"
      ];

      const buttonTag = document.getElementById("myButton");

      buttonTag.addEventListener("click", function(event) {
        const item = document.getElementById("name").value;
        

        if (!item) {
          // if no name is passed
          console.log("Enter a name");
        } else {
          const randomNumber = Math.floor(Math.random() * spiritAnimal.length);
          const randomElement = spiritAnimal[randomNumber];
          document.getElementById("animal").innerHTML =
            item + " " + "-" + " The " + " " + randomElement;
        }
      });
    </script>
  </body>
</html>

Reply:
I have pasted the formatted file and removed alert and used const keyword

Reply:
nice, don't forget to update the PR :)